components/feature/addons/src/main/java/mozilla/components/feature/addons/ui/PermissionsDialogFragment.kt

```diff
@@ -165,8 +179,11 @@ internal class PermissionsDialogFragment : AppCompatDialogFragment() {
         return rootView
     }
 
-    private fun buildPermissionsText(): String {
-        var permissionsText = getString(R.string.mozac_feature_addons_permissions_dialog_subtitle) + "\n\n"
+    @VisibleForTesting
```

Comment:
nit: @VisibleForTesting(otherwise = VisibleForTesting.PRIVATE)?

Reply:
I think `PRIVATE` is the default value :)

![image](https://user-images.githubusercontent.com/773158/72284829-e1449900-360f-11ea-9ffe-c631e0d6b801.png)